dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -490,8 +503,18 @@ public List<File> getDownloadedEsas(List<String> featureList) {
         return esaFiles;
     }
 
+    public List<File> getDownloadedAscs() {
+        List<File> ascFiles = new ArrayList<>();
```

Comment:
This entire method could be compressed into a lambda function. Here's an example: 
`return downloadedFiles.stream().filter(item->item.getName().endsWith(".asc")).collect(Collectors.toList())`

You don't need to make any change here but just some food for thought on using the Streams API to save yourself some time. 

The real reason for this comment is --> Do we need to send the file name to lowercase before checking it's extension? ie
```
f.getName().toLowercase().endsWith(".asc")
```

I'm just thinking of a case where some user has a file like `file.ASC`